OrbitGl/App.cpp

```diff
@@ -533,38 +537,40 @@ ErrorMessageOr<void> OrbitApp::ReadPresetFromFile(const std::string& filename,
     return ErrorMessage("Error opening the file for reading");
   }
 
-  try {
-    cereal::BinaryInputArchive archive(file);
-    archive(*preset);
-    file.close();
-    return outcome::success();
-  } catch (std::exception& e) {
-    ERROR("Loading preset from \"%s\": %s", file_path, e.what());
-    return ErrorMessage(
-        absl::StrFormat("Error reading the preset: %s", e.what()));
+  {
+    SCOPE_TIMER_LOG(absl::StrFormat("Read preset from \"%s\"", file_path));
+    if (!preset_info->ParseFromIstream(&file)) {
+      ERROR("Loading preset from \"%s\" failed", file_path);
+      return ErrorMessage(absl::StrFormat("Error reading the preset"));
+    }
   }
```

Comment:
Nit: I don't think that extra scope is needed. The measured difference will be lost in the noise.

Reply:
Yes, I see... removed it then.